### ISSUE TITLE
Failures in remote pylint: `Run pylint --rcfile=tests/pylintrc ` 

### ISSUE THREAD

**Issue by mxochicale** [SciKit-Surgery/scikit-surgeryvtk#204]
pylint sometimes produces the following error which might be related to pylint version but not sure about the actual cause. 

```
Run pylint --rcfile=tests/pylintrc --extension-pkg-whitelist=vtk,cv2,PySide6 sksurgeryvtk
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.7.15/x[6](https://github.com/SciKit-Surgery/scikit-surgeryvtk/actions/runs/4075429708/jobs/7021808369#step:9:7)4/bin/pylint", line 8, in <module>
    sys.exit(run_pylint())
  File "/opt/hostedtoolcache/Python/3.[7](https://github.com/SciKit-Surgery/scikit-surgeryvtk/actions/runs/4075429708/jobs/7021808369#step:9:8).15/x64/lib/python3.7/site-packages/pylint/__init__.py", line 36, in run_pylint
    PylintRun(argv or sys.argv[1:])
  File "/opt/hostedtoolcache/Python/3.7.15/x64/lib/python3.7/site-packages/pylint/lint/run.py", line 214, in __init__
    score_value = linter.generate_reports()
  File "/opt/hostedtoolcache/Python/3.7.15/x64/lib/python3.7/site-packages/pylint/lint/pylinter.py", line 1149, in generate_reports
    sect = self.make_reports(self.stats, previous_stats)
  File "/opt/hostedtoolcache/Python/3.7.15/x64/lib/python3.7/site-packages/pylint/reporters/reports_handler_mix_in.py", line 76, in make_reports
    r_cb(report_sect, stats, old_stats)
  File "/opt/hostedtoolcache/Python/3.7.15/x64/lib/python3.7/site-packages/pylint/checkers/imports.py", line 9[8](https://github.com/SciKit-Surgery/scikit-surgeryvtk/actions/runs/4075429708/jobs/7021808369#step:9:9)1, in _report_external_dependencies
    dep_info = _make_tree_defs(self._external_dependencies_info().items())
  File "/opt/hostedtoolcache/Python/3.7.15/x64/lib/python3.7/site-packages/pylint/checkers/imports.py", line 17[9](https://github.com/SciKit-Surgery/scikit-surgeryvtk/actions/runs/4075429708/jobs/7021808369#step:9:10), in _make_tree_defs
    node[1] += files
TypeError: 'tuple' object does not support item assignment
Error: Process completed with exit code 1.
```
See remote log https://github.com/SciKit-Surgery/scikit-surgeryvtk/actions/runs/4073931963/jobs/7018448358

Notes from https://github.com/SciKit-Surgery/scikit-surgeryvtk/issues/203#issuecomment-1412276985: 
* > pylint>=2.15.10 works well for different OS (Ubuntu, mac, windows) and py versions (py37, p38): https://github.com/SciKit-Surgery/scikit-surgeryvtk/actions/runs/4066002342/jobs/7001476954.
Note. You might be careful with the selection of pylint version https://github.com/PyCQA/pylint/tags as migth brake your cis (for this one I tried 2.15.0, 2.15.1, 2.15.2,2.15.3,2.15.4,2.15.5,2.15.6,2.15.7,2.15.8, and 2.16.0)

**Comment by thompson318** [comment]
This has just started happening to me today see https://github.com/SciKit-Surgery/scikit-surgeryarucotracker/issues/53. For a temporary fix I just did pylint<2.16

**Comment by mxochicale** [comment]
remote unit tests are working well with `pylint<=2.16.1` https://github.com/SciKit-Surgery/scikit-surgeryvtk/actions/runs/4122718710
